openshift-201/image-management.md

```diff
@@ -0,0 +1,434 @@
+# Best Practices of Image Management
+
+## Objectives:
+
+After completing this section, you should have an understanding of the best practices around image management.
+
+## Building Images
+Source-to-image (S2I) is a framework that makes it easy to write images that take application source code as an input and produced a new image that runs the assembled application as output.
+
+The main advantage of using S2I is the ease of use for developers.  OpenShift provides base images for the following:
+* .NET
+* Java
+* Go
+* Node.js
+* Perl
+* PHP
+* Python
+* Ruby
+
+In the following exercise, you will manage application builds with OpenShift, using the source strategy with a Git input source.
+
+The following commands are used to create a new application.  The `myapp` application created is a simple Java Sprint Boot app that will display a message based on environment variables.
+
+### Create a new application 
+```bash
+oc new-app --name myapp \
+-i redhat-openjdk18-openshift:1.8 \
+ https://github.com/BCDevOps/devops-platform-workshops \ 
+ --context-dir=openshift-201/materials/image-management/sample-app
```

Comment:
```suggestion
oc new-app --name myapp-shelly \
-i redhat-openjdk18-openshift:1.8 \
--context-dir=openshift-201/materials/image-management/sample-app \
https://github.com/BCDevOps/devops-platform-workshops#image-management
```

Reply:
better put the URL at the end to prevent issue reading context-dir option

Reply:
Fixed